examples_gbs/run_tutorial_dense.py

```diff
@@ -0,0 +1,120 @@
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports
+"""
+Dense Subgraph Tutorial
+=========================
+Graphs can be used to model a wide variety of objects: social networks, financial markets,
+biological networks, and many others. A common problem of interest is to find subgraphs that
+contain a large number of connections between its nodes. These may correspond to communities in
+social networks, correlated assets in a market, or mutually influential proteins in a biological
+network. This tutorial studies how GBS can be used to identify dense subgraphs.
+
+Mathematically, this task is known as the `dense subgraph problem
+<https://en.wikipedia.org/wiki/Dense_subgraph>`_. The density of a graph is equal to the number of
+edges in the subgraph divided by the maximum possible number of edges. Identifying the densest
+graph of a given size, known as the densest-*k* subgraph problem, is `NP-Hard
+<https://en.wikipedia.org/wiki/NP-hardness>`_.
+
+
+As shown in :cite:`arrazola2018using`, a defining feature of GBS is that when we encode a graph
+into a GBS device, it samples dense subgraphs with high probability. This property can be
+used to find dense subgraphs by sampling from a GBS device  and postprocessing the outputs.
+Let's take a look!
+
+Finding dense subgraphs
+-----------------------
+As usual, the first step is to import all required modules. We'll need the :mod:`~.gbs.data`
+module to load pre-generated samples, the :mod:`~.gbs.sample` module to postselect samples, the 
+:mod:`~.gbs.subgraph` module to search for dense subgraphs, and the :mod:`~.gbs.plot` module to 
+visualize the graphs. We'll also use Plotly for plotting and NetworkX for graph
+operations.
+"""
+from strawberryfields.gbs import data, sample, subgraph, plot
+import plotly
+import networkx as nx
+
+##############################################################################
+# In this tutorial, we'll study a 30-node graph with a planted 10-node graph, as considered in
+# :cite:`arrazola2018using`. The graph is generated by joining two Erdős–Rényi random graphs. The
+# first graph of 20 nodes is created with edge probability of 0.5. The second planted
+# graph is generated with edge probability of 0.875. The planted nodes are the last ten nodes in the
+# graph. The two graphs are joined by selecting 8 vertices at random from both graphs and adding an

```

Comment:
```suggestion
# graph. The two graphs are joined by selecting 8 nodes at random from both graphs and adding an
```
This was never officially decided, but should we stick with "nodes" over "vertices"?

Reply:
Yes, let's stick with nodes